src/core/commandRunner/CommandRunner.ts

```diff
@@ -155,6 +155,21 @@ export default class CommandRunner {
     }
   }
 
+  async showUpdateExtensionErrorMessage(err: OutdatedExtensionError) {
+    const item = await vscode.window.showErrorMessage(
+      err.message,
+      "Check for updates"
+    );
+
+    if (item == null) {
+      return;
+    }
+
+    await vscode.commands.executeCommand(
```

Comment:
This is basically an auto-upgrade? Do we require a restart?

Reply:
I don't think so, I believe it just checks for upgrades, and the user can decide what to do.  Cursorless shouldn't require a restart on upgrade